cluster-up/cluster/kind-k8s-sriov-1.17.0/provider.sh

```diff
@@ -11,11 +11,6 @@ function set_kind_params() {
 }
 
 function up() {
-    if [[ "$KUBEVIRT_NUM_NODES" -ne 2 ]]; then
```

Comment:
How is this removal related to this commit?

Reply:
Its not related I will bring it back sorry